src/Logger/WatchdogLogger.php

```diff
@@ -18,7 +19,22 @@
 class WatchdogLogger extends LoggerBase implements LoggerInterface {
 
   /**
+   * The Drupal7 service.
+   *
+   * @var \Drupal\service_container\Legacy\Drupal7
+   */
+  protected $drupal7;
+
+  /**
+   * {@inheritdoc}
```

Comment:
This is because we are using the watchdog function from D7 in the ::log() method. Should I not use it ?